src/applications/vaos/components/AppointmentTasks.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getDaysRemainingToFileClaim } from '../utils/appointment';
+import {
+  selectAppointmentTravelClaim,
+  selectIsPast,
+  selectIsInPerson,
+  selectIsClinicVideo,
+} from '../appointment-list/redux/selectors';
+import Section from './Section';
+
+export default function AppointmentTasks({ appointment }) {
+  const isPastAppointment = selectIsPast(appointment);
+  if (!isPastAppointment) return null;
+
+  const isInPerson = selectIsInPerson(appointment);
+  const isClinicVideo = selectIsClinicVideo(appointment);
+  // If it's not an in-person appointment or a clinic video appointment, don't show the link to file a claim
+  if (!isInPerson && !isClinicVideo) return null;
+
+  const claimData = selectAppointmentTravelClaim(appointment);
+  // if there is no claim data or the claim data is not successful or the claim has already been filed, don't show the link to file a claim
+  if (
+    !claimData ||
+    !claimData.metadata.success ||
+    (claimData.metadata.success && claimData.claim)
+  )
+    return null;
+
+  const daysRemainingToFileClaim = getDaysRemainingToFileClaim(
+    appointment.start,
+  );
+  if (daysRemainingToFileClaim < 1) return null;
+
+  return (
+    <Section heading="Appointment tasks">
+      <va-link-action
+        data-testid="file-claim-link"
+        className="vads-u-margin-top--1"
+        href={`/appointments/claims/?date=${appointment.start}`}
```

Comment:
Is there an issue to track updating this once the endpoint is finalized? I want to make sure we don't lose track of it.

Reply:
I just created one https://github.com/department-of-veterans-affairs/va.gov-team/issues/97545